src/css/components/_progress.scss

```diff
@@ -27,6 +27,12 @@
 /* We need an increased hit area on hover */
 .video-js .vjs-progress-control:hover .vjs-progress-holder { font-size: 1.666666666666666666em }
 
+/* Also show the current time tooltip */
+.video-js .vjs-progress-control:hover .vjs-play-progress:after {
+  display: block;
+  font-size: 0.6em;
```

Comment:
Well it still grows a tiny bit, it's just a smaller percentage of the bigger progress holder font-size.

But yes, you're right. There's definitely a hitch when you roll off. Bummer.

![hitch](http://g.recordit.co/GCkpB8scXH.gif)